docs/guides/tfhers.md

```diff
@@ -0,0 +1,364 @@
+# TFHE-rs Compatibility
```

Comment:
Hi @youben11 
Before moving on to language review and revise, I suggest clearing out the structure first. 
Instead of having  1 long doc with TOC, usually we can break them into different mds under a subgroup. (such as  "Optimization" in Guides)
So here's my proposition 

Structure 
* [TFHE-rs compatibility](docs/guides/tfhers/overview.md)
    * [Scenario 1: Shared secret key](docs/guides/tfhers/shared-key.md)
    * [Scenario 2: Pregenerated TFHE-rs keys](docs/guides/tfhers/pregenerated-key.md)
    * [Serialization of Ciphertexts and Keys](docs/guides/tfhers/serialization.md)

Could you please first break this doc into different mds? I have some more comments on the content and I'll address them on each of the mds after.

Thank you!

Reply:
done